tests/Auth/AuthDatabaseTokenRepositoryTest.php

```diff
@@ -13,14 +13,14 @@
 
 class AuthDatabaseTokenRepositoryTest extends TestCase
 {
-    public function setUp()
+    public function setUp(): void
```

Comment:
As we are doing such big refactoring, probably it's a good moment to make all `setUp` and `tearDown` methods protected as they are in PHPUnit. Now some of them are public and others are protected and there is no point to make them public.

Reply:
I realized I forgot to comment here but I've updated everything as such. Thanks for mentioning 👍